app/Http/Controllers/ScreenshotController.php

```diff
@@ -12,30 +12,30 @@
 
 class ScreenshotController extends Controller
 {
-    public function show($gameid, $screenid){
+    public function show($gameid, $screenid) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->